pkg/test/istio.io/tasks/security/authz-http/verify-lib.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+set -e
+set -u
+set -o pipefail
+
+REPEAT=${REPEAT:-100}
+THRESHOLD=${THRESHOLD:-20}
+
+ingress_url="http://istio-ingressgateway.istio-system/productpage"
+sleep_pod=$(kubectl get pod -l app=sleep -n default -o 'jsonpath={.items..metadata.name}')
+
+# verify calls curl to send requests to productpage via ingressgateway.
+# - The 1st argument is the expected http response code
+# - The remaining arguments are the expected text in the http response
+# Return 0 (success) if both the code and text is found in the response for continuously $THERESHOLD times.
+#
+# Examples:
+# 1) Expect http code 200 and "reviews", "ratings" in the body: verify 200 "reviews" "ratings"
+# 2) Expect http code 403 and "RBAC: access denied" in the body: verify 200 "RBAC: access denied"
+# 3) Expect http code 200 only: verify 200
+function verify {
+  lastResponse=""
+  wantCode=$1
+  shift
+  wantText=("$@")
+  goodResponse=0
+
+  for ((i=1; i<="$REPEAT"; i++)); do
+    response=$(kubectl exec ${sleep_pod} -c sleep -n "default" -- curl ${ingress_url} -s -w "\n%{http_code}\n")
```

Comment:
If `kubectl` or `curl` fails, it will exit the script because of `set -e`.
You need to `set +e` before and `set -e` again after.

Reply:
Thanks!